apps/server/src/modules/provisioning/strategy/tsp/tsp.jwt.payload.ts

```diff
@@ -0,0 +1,35 @@
+import { IsString, IsOptional, IsArray } from 'class-validator';
+import { JwtPayload } from 'jsonwebtoken';
+
+export class TspJwtPayload implements JwtPayload {
+	@IsString()
+	public sub!: string;
+
+	@IsOptional()
+	@IsString()
+	public sid!: string | undefined;
+
+	@IsOptional()
+	@IsString()
+	public ptscListRolle!: string | undefined;
+
+	@IsOptional()
+	@IsString()
+	public personVorname!: string | undefined;
+
+	@IsOptional()
+	@IsString()
+	public personNachname!: string | undefined;
+
+	@IsOptional()
+	@IsString()
+	public ptscSchuleNummer!: string | undefined;
+
+	@IsOptional()
+	@IsArray()
+	public ptscListKlasseId!: [] | undefined;

```

Comment:
The `!` can be removed for all optional properties.

Reply:
done